client/components/MusicWidget.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useRef, useState } from 'react';
+// import YouTubePlayer from './YoutubePlayer';
+import Example from './YoutubePlay';
+
+export default function MusicWidget() {
+  const widgetRef = useRef(null);
+  const [videoId, setVideoId] = useState('jfKfPfyJRdk');
+  // pass this into form and video and stuff as prop
+
+  function widgetMovement(e) {
```

Comment:
Are you going to have to put this movement code into every one of your widgets? When you get to the point of doing the next one, see if you can figure out how to avoid duplicating this code. Once you've give it some thought, let me know what you come up with.